cmake/onnxruntime_unittests.cmake

```diff
@@ -118,11 +114,19 @@ if(NOT onnxruntime_MINIMAL_BUILD)
     "${TEST_SRC_DIR}/framework/*.h"
     "${TEST_SRC_DIR}/platform/*.cc"
     )
+
 else()
+
   # TODO: Add tests that can be used in a minimal build
   set(onnxruntime_test_framework_src_patterns
     "${TEST_SRC_DIR}/platform/*.cc"
   )
+
+  file(GLOB onnxruntime_test_ir_src CONFIGURE_DEPENDS
+  "${TEST_SRC_DIR}/ir/*.cc"
+  "${TEST_SRC_DIR}/ir/*.h"
+  )
+
```

Comment:
Please exclude this in a minimal build as it requires OpSchema support which is not available in that build type

Reply:
Done.

Reply:
I think we want the IR tests in a full build, and in just a reduced ops build. If it's a minimal build we cannot include them. So within this 'else()' check if it's not a minimal build before including. 

---
In reply to: [479836126](https://github.com/microsoft/onnxruntime/pull/4917#discussion_r479836126) [](ancestors = 479836126)